### PR TITLE
Groovy enum to string co-ersion is not as cool as I thought, added tes…

### PR DESCRIPTION
**Pull request by imuchnik**
…t. Best I can do while out on vacation. Hope you can forgive me.

### CODE REVIEW

**Comment by imuchnik** [comment]
@vickumar1981 @sleitner @OrlandoSoto Please review. :)

**Comment by OrlandoSoto** [comment]
so in order for this to work, each jenkins instance has to have an environmental variable "ENVIRONMENT" already set?

**Comment by imuchnik** [comment]
@OrlandoSoto Yes, that is correct.

**Comment by OrlandoSoto** [comment]
@imuchnik this looks good to me, I can probably merge it then pull it down and try it out locally, however this morning @vickumar1981 expressed a concern that this PR may need re-basing?

File: jobs/jac.groovy [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    steps{
+        gradle {
+            tasks('groovydoc')
+            useWrapper()
+        }
+        shell("""
+            git add -A
+            git commit -m'Jenkins autogenerate docs'
```
Comment (vccabral) [inline jobs/jac.groovy:18]: As a suggestion, a version number or a tag might be a plus.

File: jobs/jac.groovy [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+       """)
+    }
+    triggers {
+        scm 'H/5 * * * *'
+    }
+    publishers {
+        git {
+            pushOnlyIfSuccess()
+            pushMerge(false)
+            forcePush(true)
```
Comment (vccabral) [inline jobs/jac.groovy:28]: Why is force push needed?

**Comment by vccabral** [comment]
@imuchnik I'm just popping in my head to get accustomed to these groovy based build processes.

File: jobs/jac.groovy (hunk 2 above)
Comment (imuchnik) [inline jobs/jac.groovy:28]: It force pushes the docs to gh-pages, to avoid code conflicts it will overwrite the docs to the latest everytime.

**Pull request merged**